test/negative/feature_requires_extension.ll

```diff
@@ -0,0 +1,39 @@
+; Check whether the translator reports an error for a module with token type
```

Comment:
Changed the test name, but left this comment on purpose as an example what the test is checking